framework/include/postprocessors/PseudoTimestep.h

```diff
@@ -0,0 +1,49 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "GeneralPostprocessor.h"
+
+/**
+ * Computes a time step size based on user-specified CFL number
+ */
+class PseudoTimestep : public GeneralPostprocessor
+{
+public:
+  PseudoTimestep(const InputParameters & parameters);
+
+  virtual void execute() override;
+  virtual void initialize() override;
+  using Postprocessor::getValue;
+  virtual Real getValue() const override;
+
+protected:
```

Comment:
These all need doxygen

Reply:
Not sure what you mean.

Reply:
Oh yes!!! I see! Fixed it now in as many as I could catch.